website/static/assets/grafana-dashboards/big_amount_validators.json

```diff
@@ -2751,7 +2751,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "Prometheus",
-      "description": "This alert is useful only for people who has automatic restart of the process after crash. \nThis can be made for example by using the agora-cl.bat file on windows with \"set PRYSM_AUTORESTART=true&\", or with docker with \"--restart always\"",
+      "description": "This alert is useful only for people who has automatic restart of the process after crash. \nThis can be made for example by using the prysm.bat file on windows with \"set PRYSM_AUTORESTART=true&\", or with docker with \"--restart always\"",
```

Comment:
We probably should rename the files named `prysm.bat` and `prysm.sh` and make them work.